Dockerfile

```diff
@@ -1,6 +1,15 @@
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
 WORKDIR /go/src/github.com/openshift/crd-schema-gen
 COPY . .
+ENV GO_PACKAGE github.com/openshift/crd-schema-gen
 RUN go build -o crd-schema-gen cmd/crd-schema-gen/main.go
-ENV APIS_DIR=vendor/github.com/openshift/api
-ENTRYPOINT ./crd-schema-gen --apis-dir $APIS_DIR
\ No newline at end of file
+
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+WORKDIR /go/src/generator
+COPY --from=builder /go/src/github.com/openshift/crd-schema-gen/crd-schema-gen /
+COPY --from=builder /go/src/github.com/openshift/crd-schema-gen/vendor/k8s.io/apimachinery /go/src/generator/vendor/k8s.io/apimachinery
+COPY --from=builder /go/src/github.com/openshift/crd-schema-gen/vendor/k8s.io/api /go/src/generator/vendor/k8s.io/api
```

Comment:
For some reason the crd-schema-gen binary needs these two dependencies to actually generate any output -- even though I'm mounting the `openshift/api` and `manifests` directories separately. @sttts could this also be related to the generator's nature as a parser, or possibly the transitive dependencies of the openshift API types that depend on the k8s api?

Reply:
how does the invocation of the container look like exactly?